### PR TITLE
Switch to groups for membership status

### PR DESCRIPTION
**Pull request by liam-middlebrook**
Update possible enumerations of membership status to reflect ldap
groupings better.

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -167,9 +167,9 @@ def me
             @positions = get_positions(ldap_conn, @user["dn"][0], @groups)
 
             status = [
-                @user["active"] != nil && @user["active"][0] != nil && @user["active"][0][0] == "1" ? :active : :not_active,
-                @user["alumni"] != nil && @user["alumni"][0] != nil && @user["alumni"][0][0] == "1" ? :alumni : :not_alumni, 
-                @user["onfloor"] != nil && @user["onfloor"][0] != nil && @user["onfloor"][0][0] == "1" ? :onfloor : :offfloor]
+                groups.include? "active" ? :active : :not_active,
+                groups.include? "current_student" ? :current_student : :alumni,
```
Comment (JDrit) [inline app/controllers/users_controller.rb]: should need the @ sign in front of group

**Comment by JDrit** [comment]
other than that fix, LGTM

**Comment by liam-middlebrook** [comment]
Fixed

**Comment by JDrit** [comment]
pulling into dev env and if that passed, then prod.

**Pull request merged**